apps/nextjs-website/src/app/auth/sign-up/page.tsx

```diff
@@ -94,13 +96,21 @@ const SignUp = () => {
   }, [router, userData.username]);
 
   const onResendEmail = useCallback(async () => {
-    await Auth.resendSignUp(userData.username);
-    setInfo({
-      message: signUp.emailSent(userData.username),
-      isError: false,
+    setLoader(LoaderPhase.LOADING);
+
+    const result = await Auth.resendSignUp(userData.username).catch(() => {
+      setLoader(LoaderPhase.ERROR);
+      return false;
     });
-    return null;
-  }, [signUp, userData.username]);
+
+    if (result) {
+      setLoader(LoaderPhase.SUCCESS);
+    }
+
+    setTimeout(() => {
+      setLoader(undefined);
+    }, 4000);
```

Comment:
Same as pervius comment
```suggestion
    setTimeout(() => {
      setLoader(undefined);
    }, RESET_SEND_EMAIL_BUTTON_AFTER_MS);
```